charts/hono/profileOpenshift.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+platform: openshift
+
+kafka:
+  podSecurityContext:
+    enabled: false
+  containerSecurityContext:
+    enabled: false
+
+  zookeeper:
+    podSecurityContext:
+      enabled: false
+    containerSecurityContext:
+      enabled: false
```

Comment:
this *profile* cannot actually be used when using `helm install` without unpacking the chart locally, can it?

Reply:
the profile can be used in combination with a released chart version.
You can use the profile with the -f option when installing the chart.
E.g. helm install -n hono -f profileOpenshift.yaml hono eclipse-iot/hono

Reply:
Sure, however, I do not believe that you can use the `-f` switch with resources that are not on your local file system, i.e. using the switch would require to first download and unpack the Hono chart, or am I mistaken?

Reply:
Sure you would need the file locally to apply it with the install command.

Reply:
Ok, so the c2e package chart does not rely on this file to be present, right?

Reply:
No it doesn't. For c2e there is a separate profile file. Only added the hono profile file because didn't want to create a separate PR for one file :)

Reply:
@sophokles73 If there are no more open points? I would merge the PR.